website/docusaurus.config.ts

```diff
@@ -138,6 +142,14 @@ const config: Config = {
               label: "Slack",
               href: "https://kubernetes.slack.com/archives/C081TMY9D6Y",
             },
+            {
+              label: "Zoom",
+              href: "https://us06web.zoom.us/j/85388697226?pwd=9Xxz1F0FcNUq8zFGrsRqkHMhFZTpuj.1",
+            },
+            {
+              label: "Agenda",
+              href: "https://docs.google.com/document/d/1GqeHcBlOw6ozo-qS4TLdXSi5qUn88QU6dwdq0GvxRz4/edit?tab=t.0#heading=h.ukbaidczvy3r",
+            }
```

Comment:
I don't think we need the Zoom info and agenda here, maybe a link to community section in the readme?